src/Raven.Server/RavenServer.cs

```diff
@@ -1314,7 +1457,7 @@ private void ListenToNewTcpConnection(TcpListener listener)
                         supported);
                 }
 
-                bool authSuccessful = TryAuthorize(Configuration, tcp.Stream, header, out var err);
+                bool authSuccessful = TryAuthorize(Configuration, tcp.Stream, header, tcpClient.Client.RemoteEndPoint.ToString(), out var err);
```

Comment:
Same here, let's avoid the string allocation if we can avoid it

Reply:
OK